index.html

```diff
@@ -259,6 +310,28 @@ <h2>Community and Contact</h2>
 			$("[data-tagbtn]").removeClass("active");
 			$("#All_tag").toggleClass("active");
         }
+        function searchForClientTag(tag)
+        {	
+			$("[client-data-tagbtn]").removeClass("active");
+			$("#"+tag+"_client_tag").toggleClass("active");
+			if(activeClientTag != tag)
+			{
+				$("[client-data-tag]:not([client-data-tag~='"+tag+"'])").hide();
+				$("[client-data-tag~='"+tag+"']").show();
+				activeClientTag = tag;
+			}
+			else
+			{
+				showAllClients();
+				activeClientTag = "";
+			}
+        }
+        function showAllClients()
+        {
+            $("[client-data-tag]").show();
+			$("[client-data-tagbtn]").removeClass("active");
+			$("#All_client_tag").toggleClass("active");
+        }
         function navigate() {
```

Comment:
Currently the nine clients take up a lot of space by default, and it's not immediately clear which of those is the one most people will want to download (= 1.10.64). I think it might be useful to call `$('#Stable_client_tag').click();` here once (on page load), so by default it's filtered to just the 1.10.64 clients.